pkgs/tools/typesetting/pdf2htmlEX/default.nix

```diff
@@ -43,5 +43,6 @@ stdenv.mkDerivation rec {
     license     = licenses.gpl3Plus;
     maintainers = [ maintainers.taktoa ];
     platforms   = with platforms; linux;
+    broken      = true; # 2018-09-08
```

Comment:
Would you mind adding the information from the issue description (link to upstream issue and AUR commit, no longer maintained) as a comment?

Reply:
honest question: isn't it sufficient to write it into the commit message (which is what I usually do) or do you prefer the information as code comment as well?:)

Reply:
I think having the information as a comment is vastly more useful. When somebody in the future wonders "why is this broken? can I maybe fix this?" they can just check the comment, read the explanation and check the linked sources. When the information is hidden in a comment message he *might* be able to find that with git-blame, but only if the code hasn't been moved around or heavily refactored since then.

I'm not exactly happy that @Mic92 merged this already, but the info in the commit message will do for now.

Reply:
Unlikely to happen for broken status, because the next change will hopefully remove the broken attribute again.